tests/unit/pandas_utils/test_lusid_pandas.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+import lusid
+import lusid.models as models
+import unittest
+from lusidtools.pandas_utils.lusid_pandas import lusid_respose_to_pd
+import pandas as pd
+
+test_instrument_response = lusid.models.instrument.Instrument(
+    lusid_instrument_id="LUID_TEST",
+    version=1,
+    name="Test LUID",
+    identifiers={},
+    state="Active",
+)
+
+holding_1 = lusid.models.portfolio_holding.PortfolioHolding(
+    cost={"amount": 549997.05, "currency": "GBP"},
+    cost_portfolio_ccy={"amount": 0.0, "currency": "GBP"},
+    holding_type="P",
+    instrument_uid="LUID_XQ6VSO8F",
+    properties={},
+    settled_units=137088.0,
+    sub_holding_keys={},
+    transaction=None,
+    units=137088.0,
+)
+
+holding_2 = lusid.models.portfolio_holding.PortfolioHolding(
+    cost={"amount": 12345.05, "currency": "GBP"},
+    cost_portfolio_ccy={"amount": 0.0, "currency": "GBP"},
+    holding_type="P",
+    instrument_uid="LUID_123",
+    properties={},
+    settled_units=1372222.0,
+    sub_holding_keys={},
+    transaction=None,
+    units=1372228.0,
+)
+
+test_holdings_response = lusid.models.versioned_resource_list_of_portfolio_holding.VersionedResourceListOfPortfolioHolding(
+    version=1, values=[holding_1, holding_2]
+)
+
+
+transaction_alias_response = lusid.models.transaction_configuration_type_alias.TransactionConfigurationTypeAlias(
+    type="Buy",
+    description="Purchase",
+    transaction_class="Basic",
+    transaction_group="Default",
+    transaction_roles="LongLonger",
+)
+
+
+class TestResponseToPandasObject(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        secrets_file = Path(__file__).parent.parent.parent.joinpath("secrets.json")
+        cls.api_factory = lusid.utilities.ApiClientFactory(
+            api_secrets_filename=secrets_file
+        )
+
+    def test_response_to_df(self):
+
+        holdings_df = lusid_respose_to_pd(test_holdings_response)
+        self.assertIsInstance(holdings_df, pd.DataFrame)
+
+    def test_instrument_response_to_series(self):
+
+        instrument_series = lusid_respose_to_pd(test_instrument_response)
+        self.assertIsInstance(instrument_series, pd.Series)
+
+    def test_transaction_alias_response_to_series(self):
+
+        transaction_alias_series = lusid_respose_to_pd(transaction_alias_response)
+        self.assertIsInstance(transaction_alias_series, pd.Series)
+
+    def test_response_to_pd_fail(self):
```

Comment:
also add a test for passing in `None` (for completeness)

Reply:
Can we please also check some of the values on the output as well?

Reply:
That's done.
New check for 
(1) None
(2) Some df values